src/styles/release-candidate/components/_overlay-check.scss

```diff
@@ -5,22 +5,23 @@
   }
 
   &__checkbox-tile-overlay {
-    &::before {
+    &:before {
       position: absolute;
       top: 0;
       left: 0;
       z-index: $check-tile-tile-overlay-index;
       width: 100%;
       height: 100%;
       content: " ";
+
       @include transition($transition-base);
-      background-image: linear-gradient(to bottom, rgba(0, 0, 0, .0001), rgba(0, 0, 0, .9));
+      background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.0001), rgba(0, 0, 0, 0.9));
```

Comment:
Why did you choose `0.0001` here instead of `0` for transparent? Is there a cross browser bug with using `0` as the alpha?

Reply:
It's a legacy scss code. But Im pretty sure that `transparent` has a better cross browser support.
So, I will change it to `linear-gradient(to bottom, transparent, rgba(0, 0, 0, 0.9));`